templates/keycloak.service.erb

```diff
@@ -5,6 +5,9 @@ After=network.target
 [Service]
 Type=idle
 #Environment="JAVA_OPTS=-Xms1024m -Xmx20480m -XX:MaxPermSize=768m"
+<% if !scope['keycloak::service_environment_file'].nil? -%>
```

Comment:
This could be simplified with just `<% if scope['keycloak::service_environment_file'] -%>`.